lib/smart_proxy_ansible/runner/ansible_runner.rb

```diff
@@ -111,6 +114,19 @@ def write_playbook
         File.write(File.join(@root, 'project', 'playbook.yml'), @playbook)
       end
 
+      def write_ssh_key
+        key_path = File.join(@root, 'env', 'ssh_key')
+        File.symlink(ForemanRemoteExecutionCore.settings[:ssh_identity_key_file], key_path)
```

Comment:
The path needs to be expanded first

```
ls -la /home/aruzicka/.foreman-ansible/d20210621-21220-1ml8nlz/env/ssh_key                                                                                                                                                                          
lrwxrwxrwx. 1 aruzicka aruzicka 27 Jun 21 13:12 /home/aruzicka/.foreman-ansible/d20210621-21220-1ml8nlz/env/ssh_key -> '~/.ssh/id_rsa_foreman_proxy'
```

Reply:
I should have been more explicit. The path the comes from the settings needs to be expanded